packages/fxa-settings/src/components/Settings/PageRecentActivity/en.ftl

```diff
@@ -1,13 +1,32 @@
-## Recent Activity
+## Recent account activity
+## All strings except title indicate an event that occurred from the user's account
+## These are displayed as a list with the date when the event occured
 
-recent-activity-title = Recent Account Activity
+recent-activity-title-v2 = Recent account activity
 
-recent-activity-account-create = Account was created
-recent-activity-account-disable = Account was disabled
-recent-activity-account-enable = Account was enabled
-recent-activity-account-login = Account initiated login
-recent-activity-account-reset = Account initiated password reset
-recent-activity-emails-clearBounces = Account cleared email bounces
+recent-activity-account-create-v2 = Account created
+recent-activity-account-disable-v2 = Account disabled
+recent-activity-account-enable-v2 = Account enabled
+recent-activity-account-login-v2 = Account login initiated
+recent-activity-account-reset-v2 = Password reset initiated
+recent-activity-emails-clearBounces-v2 = Email bounces cleared
```

Comment:
The previous version of this string had some questions from localizers about what "bounces" were. Could you add a comment to explain what this status means?

Reply:
Good call! I see that there's a pinned comment on Pontoon with a great explanation in the previous version of this string. @bcolsson  Is this something that we can carry over? If not I might just add that one in the ftl file.

https://pontoon.mozilla.org/fr/firefox-accounts/settings.ftl/?string=268226

Reply:
I'd have to copy paste and make a new comment. I'm okay if you want to copy the comment wholesale (though the image would be out of date, so maybe leave that out).

Reply:
Sounds good! I've moved in the relevant comment. I think all fixes are in and ready for final review @bcolsson  👍